### PR TITLE
Update contribution document

### PR DESCRIPTION
**Pull request by tigercosmos**
Add more information for installing packages.

### CODE REVIEW

**Review by Dimi1010 — APPROVED** [review]
LGTM

File: CONTRIBUTING.md
Comment (seladb) [inline CONTRIBUTING.md:32]: Let's wait until we merge [this PR](https://github.com/seladb/PcapPlusPlus/pull/1584) to see if we need it

Comment (tigercosmos) [reply CONTRIBUTING.md:32]: @seladb pre-commit will run ruff in a venv, but I think users will also run ruff without using pre-commit.

Comment (seladb) [reply CONTRIBUTING.md:32]: But we ask users to install `pre-commit` that includes `ruff`, do we really need this redundancy?

Comment (tigercosmos) [reply CONTRIBUTING.md]: Sure, removed ruff here.

**Review by seladb — APPROVED** [review]

**Pull request merged**
